rest/data/interface.go

```diff
@@ -81,6 +81,7 @@ type Connector interface {
 	FindProjects(string, int, int, bool) ([]model.ProjectRef, error)
 	// FindProjectByBranch is a method to find the projectref given a branch name.
 	FindProjectByBranch(string) (*model.ProjectRef, error)
+	GetProjectWithCommitQByOwnerRepoAndBranch(string, string, string) (*model.ProjectRef, error)
```

Comment:
can we consistently refer to the commit queue as CommitQueue rather than CommitQ

Reply:
done.
Sorry: this was left over from the cleanup and I copy/pasted it.